common/web/gesture-recognizer/src/test/auto/headless/gesturePath.spec.ts

```diff
@@ -19,7 +19,7 @@ describe("GesturePath", function() {
   describe("canary tests: serialization & deserialization", function() {
     it('from fixture', () => {
       let testJSONtext = fs.readFileSync(`${SEGMENT_TEST_JSON_FOLDER}/simple_ne_move.json`);
```

Comment:
Adding charset seems better than reading as a buffer

```suggestion
      let testJSONtext = fs.readFileSync(`${SEGMENT_TEST_JSON_FOLDER}/simple_ne_move.json`, 'utf-8');
```